alteryx/woodwork
# v0.22.0

v0.22.0 March 13, 2023

* Enhancements
    * Improved inference for numeric logical types to handle incoming ``object`` dtype data (#1645)
    * Updated datetime format inference to handle years represented by 2 digits (#1632)
    * Updated ``dependence_dict`` to handle boolean columns (#1652)
* Changes
    * Pin for jupyter-client to 7.4.9 for documentation (#1624)
    * Remove jupyter-client documentation requirement (#1627)
    * Separate Makefile command for core requirements, test requirements and dev requirements (#1658)
* Testing Changes
    * Add ruff for linting and replace isort/flake8 (#1614)
    * Specify black and ruff config arguments (#1620)
    * Add codecov token for unit tests workflow (#1630)
    * Add GitHub Actions cache to speed up workflows (#1631)
    * Add pull request check for linked issues to CI workflow (#1633 #1636)
    * Run lint fix on latest dependency update pull requests (#1640 #1641)
